scanapp/tasks.py

```diff
@@ -57,6 +63,44 @@ def scan_code_async(url, scan_id, path, file_name):
 
 
 @app.task
+def scan_code_async_final(url, scan_id, path):
+    """
+    Create and initialise the git repository at a certain path and clone the git repo using the 'url'
+    and then get the scan done asynchronously.
+    """
+    logger = logging.getLogger(__name__)
+    logger.info('git repo detected')
+
+    home_path = expanduser("~")
+    os.chdir(home_path)
+
+    os.chdir(path)
+
+    print os.curdir
+
+    clean_url = ''.join(e for e in url if e.isalnum())
```

Comment:
I am trying to give the directory a name using it's url. So cleaning the scanned URL and using it as a name.